src/Sensor/CanListenerAccessories.h

```diff
@@ -0,0 +1,93 @@
+#ifndef CAN_ACCESSORIES_LISTENER
+#define CAN_ACCESSORIES_LISTENER
+
+#include <map>
+#include <array>
+
+#include "can.h"
+#include "CanListener.h"
+#include "CanInterface.h"
+#include "IntervalCommand.h"
+
+using namespace can;
+
+/**
+ * @brief Class which stores and updates status for Can accessories
+ * 
+ */
+class CanListenerAccessories : public CanListener {
+	public:
+		/**
+		 * @brief return values for getStatus methods
+		 */
+		enum Status { Off = 0, On = 1, Unknown = 2 };
+
+		/**
+		 * @brief Constructor for CanListenerAccessories
+		 * 
+		 * @param canInterface the can interface which will be reading data from Can buffer
+		 * @param id the id of the Can message this object will listen for
+		 * @param ids the individual ids of the Can accessories whose status we want (0xFF == unused)
+		 * @note size of ids array can be increased if we need to listen to more Can status messages
+		 */
+		typedef std::array<uint8_t, 7> StatusIds;
+		CanListenerAccessories(CanInterface* canInterface, uint16_t id, StatusIds ids);
+
+		/**
+		 * @brief Nothing to handle here
+		 */
+		void handle() { }
+
+		/**
+		 * @brief Get the string name of this object
+		 */
+		String getHumanName();
+		
+		/**
+		 * @brief Get the status of headlights
+		 */
+		int getStatusHeadlights();
```

Comment:
I think we should change these to return a String, in line with other sensor getters. Or we should discuss as a team if we want to move away from only using Strings (maybe Strings make sense for floats, but would save us some characters for ints like these).

Reply:
You're right, I was thinking that using ints could save space in the json string buffer.  We haven't really stress-tested the logging system to see how fast the buffer fills up when we're gathering all the data we need.  Strings vs ints might not make that big of a difference in the end, but maybe it would be worth waiting to see?

I'll submit this for now.  I'll change it later if we all decide that Strings are the way